Dev-Corner/slides/tool_integration.html

```diff
@@ -45,65 +45,1274 @@
+
+                            - [Documentation](https://planemo.readthedocs.io/en/latest/)
+                            - [Tutorial](https://planemo.readthedocs.io/en/latest/writing_standalone.html)
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                              ### Questions
+
+                              - How to "wrap" / interface a tool for Galaxy
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Objectives
+
+                            - Discover what is a wrapper and its structure
+                            - Use the Planemo utilities to develop a good wrapper
+                            - Deal with the dependencies
+
+                        </script>
+                    </section>
+                </section>
+
+                <!-- WRAPPER -->
+                <section>
+                   <section data-markdown>
+                       <script type="text/template">
+                          ## Wrapper
+
+                       </script>
+                   </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Wrapper
+
+                            <img src="../images/wrapper_big_picture_1.png" height=400/>
+
+                            <small>[Galaxy Tool XML File](https://docs.galaxyproject.org/en/latest/dev/schema.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Wrapper
+
+                            <img src="../images/wrapper_big_picture_2.png" height=400/>
+
+                            <small>[Galaxy Tool XML File](https://docs.galaxyproject.org/en/latest/dev/schema.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Wrapper
+
+                            <img src="../images/wrapper_big_picture_3.png" height=400/>
+
+                            <small>[Galaxy Tool XML File](https://docs.galaxyproject.org/en/latest/dev/schema.html)</small>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### `command`
+                            #### How to invoke the tool?
+
+                            ```xml
+                            <command><![CDATA[
+                                graphlan.py
```

Comment:
Yes, we discussed about that. But since `seqtk` is used for the practice part, I didn't want to give the answer. I believe that it doesn't really matter, we only spend a few second on this kind of slide. They don't really have to know the tool under the example.